lib-compose/src/main/java/com/what3words/components/compose/maps/state/W3WMapState.kt

```diff
@@ -45,13 +45,24 @@ data class W3WMapState(
 
     val listMarkers: ImmutableMap<String, ImmutableList<W3WMarker>> = persistentMapOf(),
```

Comment:
I do think we move this map into the MapManager, and the W3WMapState only contains a list of markers